src/gui/dispersionreticle/translations/translations_zh_cn.json

```diff
@@ -35,33 +35,33 @@
   "focusedReticle.header": "最小落弹环",
   "focusedReticle.body": [
     "在显示标准瞄准环的同时，显示出最小的缩圈大小。\n",
-    "Uses builtin game logic to do so.\n",
-    "Other mods might influence the look of this reticle.\n"
+    "使用游戏机制实现。\n",
+    "同时使用其他类似模组可能影响显示效果。\n"
   ],
   "focusedReticle.note": [
     "在打开了插件中的服务器准星选项时最小缩圈将显示在客户端准星上。\n\n"
   ],
-  "focusedReticleExtended.label": "Focused reticle extended",
-  "focusedReticleExtended.header": "Focused reticle extended",
+  "focusedReticleExtended.label": "最小落弹环外观配置",
+  "focusedReticleExtended.header": "最小落弹环外观配置",
   "focusedReticleExtended.body": [
-    "Adds configurable reticle displaying fully-focused dispersion to in-game reticle.\n",
-    "Other mods does not have influence on the look of this reticle."
+    "这里可以配置最小落弹环的外观。\n",
+    "同时使用其他类似模组可能影响显示效果。"
```

Comment:
Other mods does not have influence of extended reticles due to how they are implemented

Update it same for extended hybrid and extended server as well

Reply:
My mistake , I though they are the same as others. How can I make changes then ? Resubmit a new one  ?